### PR TITLE
chore: update and rename config.yaml to config.yml

### PR DESCRIPTION
**Pull request by MuckT**
### Description

Minor fix to issue-templates. Had the config saved as `.yaml` instead of `.yml`. Also allowed blank issues for until a new feature template is created.

### Tested

N/A - Templates cannot be viewed on branches

### How others should test

N/A - No testing needed by QA; however, feedback on new templates will be gathered and implemented at a later date.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -26,4 +26,4 @@ contact_links:
     about: Interesting in working at cLabs? Apply here!
```
Comment (etuleu) [inline .github/ISSUE_TEMPLATE/config.yml:26]: Not part of the PR but probably should be "Interested" ?

**Review by etuleu — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (MuckT) [reply .github/ISSUE_TEMPLATE/config.yml]: Fixed in commit [334e116](https://github.com/celo-org/wallet/pull/298/commits/334e116b3816197be2d8099e360a1e87e6a1da27)
